app/src/main/res/values/strings.xml

```diff
@@ -594,4 +592,5 @@
     <string name="choose_instance_prompt">Choose an instance</string>
     <string name="app_language_title">App language</string>
     <string name="systems_language">System default</string>
+    <string name="dynamic_seek_duration_description">%s seconds</string>
```

Comment:
This is not a proper value for seconds, it should be using plurals.

To save yet another translation string, maybe we can use a format like "Seconds: %s", or even just the number.

Reply:
It's always plural: 5, 10, 15, 20, 25 seconds are all plurals.

I disagree with only number, the result would be <img src="https://user-images.githubusercontent.com/58657617/73595092-6a166c80-450c-11ea-94af-360238c1a54e.png" width="200">, then:

``Fast forward / rewind seek duration``
- 5
- 10
- 15
…
But 5 what? Seconds, minutes, years? With only number, the user don't know.

And same with Seconds: %s, to me with ``Fast forward / rewind seek duration`` it feels weird, it don't feel like a sentence. If you watch the rest of the menus, ans list preferences, it stays natural, like a sentence.

Reply:
Well I agree with the mauricio. Some numbers might need special treatment in some languages.
![image](https://user-images.githubusercontent.com/17365767/74614077-0c417180-5115-11ea-96e2-34192bc37838.png)

Reply:
> It's always plural: 5, 10, 15, 20, 25 seconds are all plurals.

Some languages have different forms depending on the quantity.

> I disagree with only number, the result would be
> But 5 what? Seconds, minutes, years? With only number, the user don't know.

True, the caption of the option would have to include the unit nonetheless.

Reply:
I use plural and included few, many, other as zero, one and two will never be used.